src/crypto/sha256_avx512.cpp

```diff
@@ -0,0 +1,350 @@
+#ifdef ENABLE_AVX512
+
+#include <stdint.h>
+#include <immintrin.h>
+
+#include <crypto/sha256.h>
```

Comment:
Why?

Reply:
Does not seem to be used sha256.h
Keep the same as other files(avx2 sse4)?